### PR TITLE
Fix PSR2

### PR DESCRIPTION
**Pull request by joaorobertopb**
**My suggestion:** Standardize the code with the PSR2 rules.

I used the following [Php-cs-fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer) commands:

- **php-cs-fixer fix . --diff --dry-run** (parse)
- **php-cs-fixer fix .** (Fix)

**P.S:** (This package does not have the _'Contributing'_ file. I was in doubt to send this pull request ...)

### CODE REVIEW

File: src/Commands/HookCommand.php [hunk 1]
```diff
@@ -1,36 +1,36 @@
...
-            ->setHelp("This command allows you to test your {$this->hook} hook")
-        ;
-    }
-
-    protected function execute(InputInterface $input, OutputInterface $output)
-    {
-        $output->writeln(shell_exec($this->script));
-    }
-}
+<?php
```
Comment (BrainMaestro) [inline src/Commands/HookCommand.php]: I can't tell what actually changes here

**Comment by BrainMaestro** [comment]
@joaorobertopb this contribution is fine. Any potential improvements are welcome

File: src/Commands/HookCommand.php (hunk 1 above)
Comment (joaorobertopb) [reply src/Commands/HookCommand.php]: The difference is in ANSI format and number of bytes:

**Before:** 947 bytes - ANSI PC
**After:** 911 bytes - ANSI UNIX

Comment (BrainMaestro) [reply src/Commands/HookCommand.php]: Okay cool

**Comment by BrainMaestro** [comment]
I'll go ahead and merge this. Would you mind add `php-cs-fixer` support to the repo? it can even be setup with githooks too 😄  and also run in travis. Thanks!

**Comment by joaorobertopb** [comment]
It's ok :smiley:! I'll do it later.

**Pull request merged**